Meta/config/Meta.xsd

```diff
@@ -30,6 +30,7 @@
 			<xs:element name="Log" type="tns:Log" minOccurs="0" 	maxOccurs="1" />
 			<xs:element name="SourceVariableThreadPool" type="tns:SourceVariableThreadPool" minOccurs="0" 	maxOccurs="1" />
 			<xs:element name="Server" type="tns:Server" minOccurs="0" 	maxOccurs="1" />
+			<xs:element name="BuildInformation" type="tns:BuildInformation" minOccurs="0" 	maxOccurs="1" />
```

Comment:
What motivates having BuildInfo in the config file schema?
To my mind it should not be there. Or?

Reply:
Yep, probably a good point. I'll try removing it (+references in the code to it). More soon...

Reply:
Done